### PR TITLE
Drop pandas

### PR DESCRIPTION
**Pull request by asvetlov**
@Axik

### CODE REVIEW

**Comment by Axik** [comment]
Huge amount of work.

File: datarobot_batch_scoring/batch_scoring.py [hunk 1]
```diff
@@ -220,41 +214,33 @@ def has_next(self):
...
+                [p['class_probabilities'][c] for c in sorted_classes]
+                for p in
+                sorted(predictions, key=operator.itemgetter('row_id'))]
     elif result['task'] == TargetType.REGRESSION:
-        pred = pd.DataFrame({pred_name: [p["prediction"] for p in
-                             sorted(predictions, key=lambda p: p['row_id'])]})
+        pred = [[p['row_id']+batch.id, p['prediction']]
+                for p in
+                sorted(predictions, key=operator.itemgetter('row_id'))]
+        out_fields = ['row_id', '']
```
Comment (Axik) [inline datarobot_batch_scoring/batch_scoring.py]: you forgot pred_name

Comment (asvetlov) [inline datarobot_batch_scoring/batch_scoring.py]: Good point. I'll add a test for the case

Comment (asvetlov) [inline datarobot_batch_scoring/batch_scoring.py]: Fixed by 507dcf6

**Pull request merged**